bin/clean_topics_vs_practices.py

```diff
@@ -0,0 +1,28 @@
+#!python3
+import json
+
+
+def main():
+    with open("config.json") as f:
+        config = json.load(f)
+
+    concepts = [c['slug'] for c in config['concepts']]
+    concepts.sort()
+
+    for practice_exercise in config['exercises']['practice']:
+        if practice_exercise['topics'] is None:
+            continue
+
+        for topic in practice_exercise['topics']:
+            if topic in concepts:
+                practice_exercise['topics'].remove(topic)
+                practice_exercise['practices'].append(topic)
+
+    with open("config.json", 'w') as f:
```

Comment:
```suggestion
    with open("config.json", 'w', encoding="utf-8") as f:
```

Let's be explicit about the encoding. It wouldn't do for someone to run this on a Windows machine and accidentally encode the file as CP-1252 or something.